alembic/env.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import with_statement
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+from logging.config import fileConfig
+import os, sys
+sys.path.append(os.getcwd())
+
+from app import db, create_app
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# app = create_app(os.environ['CONFIG_ENV'])
```

Comment:
we should consider having a .env file and have a startup script set the environment variables in dev mode